csv2ofx/utilz/csvtrim

```diff
@@ -6,74 +6,337 @@
 #
 # TO-DO: integrate in csv2ofx?
 ################################################################################
-[ "$DEBUG" ] && set -x
+[[ "$CSVTRIM_TRACE" ]] && set -x
 set -o pipefail
+shopt -s expand_aliases
 
 __author__='Marco "sphakka" Poleggi'
+__version__='0.1.2'
```

Comment:
Where does this version number come from?

Reply:
Oh, I see you're the author of the script. Okay. Let's accept this as the author's choice.